doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -1541,13 +1541,19 @@ whose general syntax is
    :name: =>
 
 .. prodn::
```

Comment:
Sorry I'm also commenting on the general shape of this chapter. After a `.. tacn::` declaration, there must be an indented body, which isn't the case here. I suggest indenting the grammar rules and the following description. The other solutions would imply moving things around and reformulating.

Reply:
I agree, but I leave this as it is for now so that checking the diff is simpler (if I indent the entire thing, then you would get a big change)

Reply:
OK